Kitodo/src/main/java/de/sub/goobi/export/download/ExportMets.java

```diff
@@ -133,200 +120,35 @@ public boolean startExport(Process myProcess, URI inZielVerzeichnis)
      * @param targetFolder
      *            the folder to prove and maybe create it
      */
-    protected URI prepareUserDirectory(URI targetFolder) {
-        User myBenutzer = (User) Helper.getManagedBeanValue("#{LoginForm.myBenutzer}");
-        if (myBenutzer != null) {
+    protected void prepareUserDirectory(URI targetFolder) {
+        User user = (User) Helper.getManagedBeanValue("#{LoginForm.myBenutzer}");
+        if (user != null) {
             try {
-                fileService.createDirectoryForUser(targetFolder, myBenutzer.getLogin());
+                fileService.createDirectoryForUser(targetFolder, user.getLogin());
             } catch (Exception e) {
                 Helper.setFehlerMeldung("Export canceled, could not create destination directory: " + targetFolder, e);
             }
         }
-        return targetFolder;
     }
 
     /**
      * write MetsFile to given Path.
      *
-     * @param myProcess
+     * @param process
      *            the Process to use
      * @param metaFile
      *            the meta file which should be written
      * @param gdzfile
      *            the FileFormat-Object to use for Mets-Writing
      */
-
-    protected boolean writeMetsFile(Process myProcess, URI metaFile, Fileformat gdzfile, boolean writeLocalFilegroup)
+    protected boolean writeMetsFile(Process process, URI metaFile, Fileformat gdzfile, boolean writeLocalFileGroup)
```

Comment:
Parameter `writeLocalFileGroup` and return value are missing in Javadoc. Can you please add them?